library/Ivoz/Provider/Domain/Model/HuntGroupsRelUser/HuntGroupsRelUser.php

```diff
@@ -3,13 +3,15 @@
 
 use Ivoz\Core\Domain\Assert\Assertion;
 use Ivoz\Provider\Domain\Model\HuntGroup\HuntGroupInterface;
+use Ivoz\Provider\Domain\Traits\RoutableTrait;
 
 /**
  * HuntGroupsRelUser
  */
 class HuntGroupsRelUser extends HuntGroupsRelUserAbstract implements HuntGroupsRelUserInterface
 {
     use HuntGroupsRelUserTrait;
+    use RoutableTrait;
```

Comment:
It whould be nice to have an abstract method here `abstract function getAvailableRouteTypes(): array` that entities implement instead of overwriting $routeTypes values

Reply:
We will implement this in #1304 for all entities implementing Routable Trait